src/main/resources/assets/js/components/DeploymentComponent.jsx

```diff
@@ -23,8 +23,10 @@ define([
       this.setState({loading: bool});
     },
 
-    handleDestroyDeployment: function () {
-      this.props.destroyDeployment(this.props.model, this);
+    handleDestroyDeployment: function(forceStop) {
+      return function() {
+        this.props.destroyDeployment(this.props.model, {forceStop: forceStop}, this);
+      }.bind(this);
```

Comment:
You need to return a function here, because the onClick-argument expects a function that can be executed as a callback, if the onClick is triggered by the user.
If you don't return a function to onClick, the destroyDeployment() gets immediately executed "()" at define state, because I am assigning the forceStop parameter with handleDestroyDeployment(true) and onClick also wouldn't have a callback.